bool brakes_on
---
bool was_set
